stable/postgresql/README.md

```diff
@@ -33,123 +33,132 @@ The command deploys PostgreSQL on the Kubernetes cluster in the default configur
-| `affinity`                 | Affinity settings for pod assignment            | {}                                                         |
-| `tolerations`              | Toleration labels for pod assignment            | []                                                         |
-| `probes.liveness.initialDelay`      | Liveness probe initial delay           | `60`                                                       |
-| `probes.liveness.timeoutSeconds`    | Liveness probe timeout seconds         | `5`                                                        |
-| `probes.liveness.failureThreshold`  | Liveness probe failure threshold       | `6`                                                        |
-| `probes.readiness.initialDelay`     | Readiness probe initial delay          | `5`                                                        |
-| `probes.readiness.timeoutSeconds`   | Readiness probe timeout seconds        | `3`                                                        |
-| `probes.readiness.failureThreshold` | Readiness probe failure threshold      | `5`                                                        |
-| `podAnnotations`           | Annotations for the postgresql pod              | {}                                                         |
-| `deploymentAnnotations`    | Annotations for the postgresql deployment       | {}                                                         |
-
-The above parameters map to the env variables defined in [postgres](http://github.com/docker-library/postgres). For more information please refer to the [postgres](http://github.com/docker-library/postgres) image documentation.
+The following tables lists the configurable parameters of the PostgreSQL chart and their default values.
+
+|         Parameter          |                Description                |                            Default                        |
+|----------------------------|-------------------------------------------|---------------------------------------------------------- |
+| `image.registry`           | PostgreSQL image registry                 | `docker.io`                                               |
+| `image.repository`         | PostgreSQL Image name                     | `bitnami/postgresql`                                      |
+| `image.tag`                | PostgreSQL Image tag                      | `{VERSION}`                                               |
+| `image.pullPolicy`         | PostgreSQL image pull policy              | `Always`                                                  |
+| `image.pullSecrets`        | Specify image pull secrets                | `nil` (does not add image pull secrets to deployed pods)  |
+| `image.debug`              | Specify if debug values should be set     | `false`                                                   |
+| `replication.enabled`      | Would you like to enable replication      | `false`                                                   |
```

Comment:
I'd enabled replication by default as we do on MariaDB.

Reply:
We are enabling it by default in `values-production.yaml`